var/spack/repos/builtin/packages/r-limma/package.py

```diff
@@ -32,7 +32,10 @@ class RLimma(RPackage):
     homepage = "https://www.bioconductor.org/packages/limma/"
     git      = "https://git.bioconductor.org/packages/limma.git"
 
+    version('3.36.2',  commit='0cd5c13e22565182226bd2937ffcf8c59de1ca59')
     version('3.34.9',  commit='6755278a929f942a49e2441fb002a3ed393e1139')
     version('3.32.10', commit='593edf28e21fe054d64137ae271b8a52ab05bc60')
 
-    depends_on('r@3.4.0:3.4.9', when='@3.32.10:')
+    depends_on('r@3.5.0:3.5.9', when='@3.36.2')
+    depends_on('r@3.4.2:3.4.9', when='@3.34.9')
+    depends_on('r@3.4.0:3.4.9', when='@3.32.10')
```

Comment:
Any idea what range of versions require which range of R versions? For example, if I add 3.35.1, which R versions are needed?

Reply:
It's not totally clear without scanning through git commits, but it seems 3.35.1 would also fall under bioconductor release 3.7, so R 3.5

Because bioconductor packages are tied to a specific bioconductor release, I'm not sure why anyone would ever want one of the 'in between' versions ( I don't think they're ever released or tagged)